bundle.go

```diff
@@ -34,6 +124,9 @@ func UploadResources(resources []interface{}, mgoSession *mgo.Session, mDB strin
 			if ok {
```

Comment:
`ok` means that the resource was successfully cast as a `Patient` object but it doesn't necessarily mean that all of the fields were deserialized correctly. For example, today we encountered an issue with `p.BirthDate` being `nil`. You really should error check that these fields are not nil before trying to access them. The extra error logging (in the cases where these fields _are_ nil) will also help us debug things more easily.

I know it's a PITA but it's totally worth it.